cpp/data/installer/ixtheo.cronjobs

```diff
@@ -33,5 +33,6 @@ LOG_DIR=/usr/local/var/log/tuefind
 0 22 * * * "$BIN/new_journal_alert" ixtheo {ixtheo_host} "IxTheo Team<no-reply@ub.uni-tuebingen.de>" "IxTheo Subscriptions" > "$LOG_DIR/new_journal_alert_ixtheo.log" 2>&1
 30 22 * * * "$BIN/new_journal_alert" relbib {relbib_host} "RelBib Team<no-reply@ub.uni-tuebingen.de>" "RelBib Subscriptions" > "$LOG_DIR/new_journal_alert_relbib.log" 2>&1
 0 23 * * * "$BIN/update_tad_email_acl.sh" "$EMAIL"
-30 23 * * * "$BIN/rss_aggregator" "/usr/local/var/lib/tuelib/rss_aggregator_relbib.conf" "$EMAIL" "/usr/local/vufind/public/relbib_docs/news.rss" > "$LOG_DIR/rss_aggregator_relbib.log" 2>&1
+30 23 * * * "$BIN/rss_aggregator" relbib "$EMAIL" "/usr/local/vufind/public/relbib_docs/news.rss" > "$LOG_DIR/rss_aggregator_relbib.log" 2>&1
+40 23 * * * "$BIN/rss_aggregator" ixtheo "$EMAIL" "/usr/local/vufind/public/ixtheo_docs/news.rss" > "$LOG_DIR/rss_aggregator_ixtheo.log" 2>&1
```

Comment:
In fact, there is no directory called "ixtheo_docs", this one is just called "docs".

Reply:
And be aware that of course the feed file can already be generated for ixtheo, but keep in mind that if you would like to use the whole functionality on tuefind side, lots of things need to be changed (e.g. the templates are made for tuefind2 and specialized for relbib2, which has no effect on ixtheo yet and might also need some changes for krimdok).

Also, if you enable the ixtheo cronjob you should also enable the krimdok cronjob.

Reply:
We need different directories for ixtheo and relbib.  I only wanted to keep the option of also using this mechanism for KrimDok.  We don't need to activate it yet, or do we?

Reply:
Removed the IxTheo entry.